.github/workflows/pushaction.yml

```diff
@@ -1,34 +1,26 @@
 name: Compile and deploy snapshot artifacts
 
-on:
-  push:
-    branches:
-      - master
-      - sdl/**
+on: [push, pull_request]
```

Comment:
This will deploy a snapshot on each PR, very dangerous :)

Reply:
That's why I added the `if: github.event_name == 'push' && github.ref == 'refs/heads/master' && github.repository_owner == 'libgdx'` to the actual publish step ;)

Reply:
Ah, great. Can you add a comment so the next one won't stumble accross this?